doc/languages-frameworks/texlive.section.md

```diff
@@ -0,0 +1,128 @@
+
+# TeX Live {#sec-language-texlive}
+
+Since release 15.09 there is a new TeX Live packaging that lives entirely under attribute `texlive`.
+
+## User's guide {#sec-language-texlive-user-guide}
+
+- For basic usage just pull `texlive.combined.scheme-basic` for an environment with basic LaTeX support.
+- It typically won't work to use separately installed packages together. Instead, you can build a custom set of packages like this:
+
+  ```nix
+  texlive.combine {
+    inherit (texlive) scheme-small collection-langkorean algorithms cm-super;
+  }
+  ```
+
+- There are all the schemes, collections and a few thousand packages, as defined upstream (perhaps with tiny differences).
+- By default you only get executables and files needed during runtime, and a little documentation for the core packages. To change that, you need to add `pkgFilter` function to `combine`.
+
+  ```nix
+  texlive.combine {
+    # inherit (texlive) whatever-you-want;
+    pkgFilter = pkg:
+      pkg.tlType == "run" || pkg.tlType == "bin" || pkg.pname == "cm-super";
+    # elem tlType [ "run" "bin" "doc" "source" ]
+    # there are also other attributes: version, name
+  }
+  ```
+
+- You can list packages e.g. by `nix repl`.
+
+  ```ShellSession
+  $ nix repl
+  nix-repl> :l <nixpkgs>
+  nix-repl> texlive.collection-[TAB]
```

Comment:
Previously, we had a tab key symbol here (though it does not seem to be properly styled in the [current stylesheet](https://nixos.org/manual/nixpkgs/stable/#sec-language-texlive-users-guide)). Not sure if we should try to add it.

Otherwise, the diff looks fine and should be ready to merge after squash.

Reply:
I think that [TAB] is an improvement over that obscure character.

Reply:
The issue is not the symbolic representation but rather a graphical one. Compare

<pre>
nix-repl> texlive.collection-<kbd>→|</kbd>
</pre>

with

```
nix-repl> texlive.collection-[TAB]
```

Reply:
Oh yeah, it looks a lot better with that styling.